packages/react/src/components/Dropdown/DropdownItem.tsx

```diff
@@ -6,11 +7,12 @@ import { Button } from '../Button/Button';
 export type DropdownItemProps = Omit<ButtonProps, 'variant' | 'size' | 'color'>;
```

Comment:
Does this omit for size still matter?

Reply:
Good point! ☺️ We should check this in all components actually - I can do a regex-search

Reply:
We decided to do this in another PR, as this is an occurence in other placess as well